server/src/com/google/refine/Refine.java

```diff
@@ -464,7 +464,7 @@ static private String fixWindowsUnicodePath(String path) {
     public void init(String host, int port) throws Exception {
 
     	if(host.contains("0.0.0.0")) {
```

Comment:
Why `contains` and not string equality here? For instance `140.0.0.0` contains `0.0.0.0` but I don't think we want to rewrite that to `localhost`.

Also, it would perhaps be marginally cleaner to only set a variable storing the cleaned host in the `if` block, and then build the URI based on that cleaned host?

Reply:
@wetneb Made the changes you requested!